### ISSUE TITLE
Can't change language data as an Administrator

### ISSUE THREAD

**Issue by RappyTV** [mirego/accent#399]
When I try to change the german slug from `de` to `de_de`:
### Payload
```json
[
    {
        "payload": {
            "data": {
                "updateRevision": null
            }
        },
        "operationName": "RevisionUpdate"
    }
]
```
### Response
```json
[{"payload":{"data":{"updateRevision":null}},"operationName":"RevisionUpdate"}]
```
But it does not update - even after refreshing.

**Comment by RappyTV** [comment]
Note that this ocurred on version 1.19.7 idk if it was fixed in 1.19.8+

**Comment by simonprev** [comment]
Oh it was a typo in the ability name. Will be fixed in the next release 👍
